const handleError = (err) => {
  // console.trace()
  // console.handleError(err.trace)
  console.error(err)
  // throw err
}

export default handleError
